development/process.md

```diff
@@ -38,6 +38,22 @@ reviewers, impact on other contributors, and to keep the amount of rework from t
     1. Make sure the pull request passes the tests in CI.
     2. If known, request a review from an expert in the area changed.  If unknown, ask for help on [Slack](/slack.html).
 
+    There are some tests that use external services, like Google BigQuery, and require
+    additional credentials. The Trino project cannot share these credentials with
+    contributors, so it runs these tests in its CI workflows only on branches in the
+    Trino repository, not in contributor forks.
+
+    Trino maintainers, so project members with write access to the repository, can
+    schedule additional workflow runs after reviewing a PR by adding a comment
+    starting with this line:
+
+    ```
+    /test-with-secrets sha=<all-40-characters>
+    ```
+
+    The SHA value should be the full 40 character git SHA of a commit
+    of the feature branch, usually the head commit.
```

Comment:
```suggestion
    The SHA value should be the full 40 character git SHA of a commit
    of the feature branch, usually the head commit. This schedules an additional workflow run with all the secrets available so that tests which require external services are also executed.
```

Reply:
This repeats what the first two paragraphs say. Is something missing there?

Reply:
The earlier paragraphs only say some tests require secrets and we don't share them with contributors or run them on fork PRs. While it implicitly means that this would run with secrets it isn't mentioned anywhere.

LGTM either way. re-reading this it's readable.